charts/sonarqube/templates/_pod.tpl

```diff
@@ -0,0 +1,472 @@
+      imagePullPolicy: {{ .Values.image.pullPolicy  }}
+      command:
+        - sh
+        - -c
+        - |
+          #!/bin/sh
+          if [ -f /tmp/props/sonar.properties ]; then
+            cat /tmp/props/sonar.properties > /tmp/result/sonar.properties
+          fi
+          if [ -f /tmp/props/secret.properties ]; then
+            cat /tmp/props/secret.properties > /tmp/result/sonar.properties
+          fi
+          if [ -f /tmp/props/sonar.properties -a -f /tmp/props/secret.properties ]; then
+            awk 1 /tmp/props/sonar.properties /tmp/props/secret.properties > /tmp/result/sonar.properties
+          fi
+      volumeMounts:
+        - mountPath: /tmp/result
+          name: concat-dir
+        {{- if or .Values.sonarProperties .Values.sonarSecretKey (not .Values.elasticsearch.bootstrapChecks) }}
+        - mountPath: /tmp/props/sonar.properties
+          name: config
+          subPath: sonar.properties
+        {{- end }}
+        {{- if .Values.sonarSecretProperties }}
+        - mountPath: /tmp/props/secret.properties
+          name: secret-config
+          subPath: secret.properties
+        {{- end }}
+      {{- with .Values.initContainers.securityContext }}
+      securityContext: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.initContainers.resources }}
+      resources: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      env:
+        {{- range (include "sonarqube.combined_env" . | fromJsonArray)  }}
+        - name: {{ .name }}
+          value: {{ .value | quote }}
+        {{- end }}
+    {{- end }}
+    {{- if .Values.prometheusExporter.enabled }}
+    - name: inject-prometheus-exporter
+      image: {{ default (include "sonarqube.image" $) .Values.prometheusExporter.image }}
+      imagePullPolicy: {{ .Values.image.pullPolicy  }}
+      {{- with (default .Values.initContainers.securityContext .Values.prometheusExporter.securityContext) }}
+      securityContext: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with (default .Values.initContainers.resources .Values.prometheusExporter.resources)}}
+      resources: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      command: ["/bin/sh", "-c"]
+      args: ["curl -s '{{ include "prometheusExporter.downloadURL" . }}' {{ if $.Values.prometheusExporter.noCheckCertificate }}--insecure{{ end }} --output /data/jmx_prometheus_javaagent.jar -v"]
+      volumeMounts:
+        - mountPath: /data
+          name: sonarqube
+          subPath: data
+      env:
+        - name: http_proxy
+          value: {{ default "" .Values.prometheusExporter.httpProxy }}
+        - name: https_proxy
+          value: {{ default "" .Values.prometheusExporter.httpsProxy }}
+        - name: no_proxy
+          value: {{ default "" .Values.prometheusExporter.noProxy }}
+        {{- range (include "sonarqube.combined_env" . | fromJsonArray)  }}
+        - name: {{ .name }}
```

Comment:
This break again from the 10.5.1 and secret values

Reply:
@jonesbusy , correct I missed this change during rebase with master branch.

Reply:
@jCOTINEAU , pls, pay attention to this thread. 
Some changes were missed from original PR.

Reply:
@davividal is working on it, we are discussing internally and will notify you soon